lib/os.js

```diff
@@ -53,6 +53,7 @@ const {
   getUserInfo,
   getUptime,
   isBigEndian,
+  getOSVersion,
```

Comment:
I think this needs to follow how `getOSRelease` is set up since the C++ side is expecting an argument in case of error.

Reply:
Thank you sr!